### PR TITLE
mastercard: update bins (4.1.1)

### PR DESCRIPTION
**Pull request by allangrds**
## Description
Criamos o PR https://github.com/pagarme/pagarme-js/pull/162 para criar a versão 2.2.2 com os novos bins da Mastercard pro checkout. Criei esse para atualizar esses bins na versão mais recente da lib.

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 4d19d8b2dc8d5a7223ed8d4b37057dd3af08dc7d
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 4d19d8b2dc8d5a7223ed8d4b37057dd3af08dc7d
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: fdd71b082197dd98fe0f3153048c6505b1c01c49
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: fdd71b082197dd98fe0f3153048c6505b1c01c49
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: f6ebfe8ee15af6e08d019da3e22fa5caaf8c09bf
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: f6ebfe8ee15af6e08d019da3e22fa5caaf8c09bf
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: bd43bf25a05aca9fc0f8928bdbde9a207e8c0f54
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: bd43bf25a05aca9fc0f8928bdbde9a207e8c0f54
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: bd43bf25a05aca9fc0f8928bdbde9a207e8c0f54
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: e4e65148e6a186445683f1ce4cb4c8cd77e3d276
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: bd43bf25a05aca9fc0f8928bdbde9a207e8c0f54
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: e4e65148e6a186445683f1ce4cb4c8cd77e3d276
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: bd43bf25a05aca9fc0f8928bdbde9a207e8c0f54
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 84bed849670eddd818265bfaa3d175646bd19390
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: bd43bf25a05aca9fc0f8928bdbde9a207e8c0f54
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 84bed849670eddd818265bfaa3d175646bd19390
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: bd43bf25a05aca9fc0f8928bdbde9a207e8c0f54
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 84bed849670eddd818265bfaa3d175646bd19390
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 9969be0eba52eace32d7e393021feba43ef34115
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: bd43bf25a05aca9fc0f8928bdbde9a207e8c0f54
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 84bed849670eddd818265bfaa3d175646bd19390
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 9969be0eba52eace32d7e393021feba43ef34115
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: bd43bf25a05aca9fc0f8928bdbde9a207e8c0f54
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 84bed849670eddd818265bfaa3d175646bd19390
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 088488f30970c205bd7bdc0a0fe771255cc75d88
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: bd43bf25a05aca9fc0f8928bdbde9a207e8c0f54
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 84bed849670eddd818265bfaa3d175646bd19390
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 088488f30970c205bd7bdc0a0fe771255cc75d88
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: bd43bf25a05aca9fc0f8928bdbde9a207e8c0f54
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 84bed849670eddd818265bfaa3d175646bd19390
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: d807ecb2945c11615ec53285d9df0da46f828af5
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 2c28dc67956fc26a6a978043b66bb6e502a8235e
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 13c1a7bdffa1e8f4efecb662e49c7457b47a6443
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: bd43bf25a05aca9fc0f8928bdbde9a207e8c0f54
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: 84bed849670eddd818265bfaa3d175646bd19390
  * Commits must be in the following format: **%{context}: %{description}**

 * Commit: d807ecb2945c11615ec53285d9df0da46f828af5
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 63a98020da185cb2c5a49601aaedaad433cf38f6
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There are some issues with your commit and our naming convention

 * Commit: 63a98020da185cb2c5a49601aaedaad433cf38f6
  * Commits must be in the following format: **%{context}: %{description}**

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
- - 4
```
Comment (allangrds) [inline .travis.yml]: https://travis-ci.org/pagarme/pagarme-js/jobs/530804888

Sobre a remoção desse cara aqui.

**Comment by rsmelo** [comment]
abri outro pr (https://github.com/pagarme/pagarme-js/pull/166) que poderia sair junto da versão `4.1.1` o que acha de não bumpar a versão nesse pr e bumpar após o approve dos dois prs?
No mais está tudo ok

**Comment by lucianopf** [comment]
> abri outro pr (#166) que poderia sair junto da versão `4.1.1` o que acha de não bumpar a versão nesse pr e bumpar após o approve dos dois prs?
> No mais está tudo ok

Boa! ⭐️ ⭐️ 

(Coisalinda ver essa sdk evoluindo!!)

**Pull request merged**